pallets/compliance-manager/src/lib.rs

```diff
@@ -735,9 +832,22 @@ impl<T: Trait> Module<T> {
                 asset_compliance_with_results.result = true;
             }
         }
+        // Aggregating the result with the implicit requirement result.
+        asset_compliance_with_results.result =
+            Self::compute_final_result(&asset_compliance_with_results);
         asset_compliance_with_results
     }
 
+    fn compute_final_result(asset_compliance: &AssetComplianceResult) -> bool {
```

Comment:
Since this function is not used anywhere else, you can do the final calculation directly in the above function.
```
if let Some(ref imp_result) = asset_compliance_with_results.implicit_requirements_result {
    asset_compliance_with_results.result &= imp_result.from_result && imp_result.to_result
}
```